lighthouse/blueprints/plates.py

```diff
@@ -81,3 +83,36 @@ def create_plate_from_barcode() -> Tuple[Dict[str, Any], int]:
     except Exception as e:
         logger.exception(e)
         return {"errors": [type(e).__name__]}, HTTPStatus.INTERNAL_SERVER_ERROR
+
+
+def format_plate(barcode: str) -> Dict[str, Any]:
+    """Used by flask route /plates to format each plate
+    Arguments:
+        barcode
+    Returns:
+        {}, HTTPStatus
+    """
+    plate_map = has_sample_data(barcode)
+    number_of_positives = count_positive_samples(barcode) if plate_map else None
+
+    return {
+        "plate_barcode": barcode,
+        "plate_map": plate_map,
+        "number_of_positives": number_of_positives,
```

Comment:
I know it was worded like this in the mock-up, but I wonder if we should try to differentiate more in the language we use between 'Positive' and 'Selected for cherrypicking' samples?

So, for instance, in this feature, we could call the column in the table, and this key in the endpoint 'To pick' or similar. What do you think?

Reply:
The create_plates endpoint also calls it number_of_positives, so think consistency within the lighthouse wins out at the moment, but I'm inclined to agree that calling them positives was a mistake.

Reply:
I guess that's because initially, they were the same thing, i.e. the picking criteria was just Result = 'Positive'

Reply:
That's true here, but we're about to call various fields 'filtered_positives' and the like... although I suppose its not too late to change that.